dht.go

```diff
@@ -65,19 +66,13 @@ type IpfsDHT struct {
 	plk sync.Mutex
 }
 
-// NewDHT creates a new DHT object with the given peer as the 'local' host
-func NewDHT(ctx context.Context, h host.Host, dstore ds.Batching, validator record.Validator) *IpfsDHT {
```

Comment:
keep those around for backwards compatibility. it is not acceptable to have to change existing working code.

Reply:
i mean the original constructor that takes a context, host, and datastore.